PipelinesTasks/PRMetrics/src/task/src/repos/iReposInvoker.d.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { CommentThreadStatus, GitPullRequestCommentThread } from 'azure-devops-node-api/interfaces/GitInterfaces'
+import PullRequestDetails from './pullRequestDetails'
+
+/**
+ * An interface for invoking repository functionality with any underlying repository store.
+ */
+export default interface IReposInvoker {
+  /**
+   * Gets a value indicating whether the current repository provides complete functionality.
+   */
+  isCommentsFunctionalityAvailable: boolean
+
+  /**
+   * Gets a value indicating whether the OAuth access token is available to the task.
```

Comment:
nit: should this be "for the task" instead maybe?

Reply:
Thanks for highlighting this. There are a few things wrong with it – it should be the "OAuth token" rather than "OAuth access token", and the standard terminology is "accessible" rather than "available". I've now rewritten it the sentence taking these points into account.